pilot/pkg/model/config.go

```diff
@@ -1059,11 +1084,26 @@ func (store *istioConfigStore) AuthenticationPolicyByDestination(hostname Hostna
 			out = spec
 		}
 	}
-	// Zero-currentMatchLevel implies no config matching the destination found.
-	if currentMatchLevel == 0 {
-		return nil
+	// Non-zero currentMatchLevel implies authentication policy was found for the given host.
+	if currentMatchLevel != 0 {
+		return &out
+	}
+
+	// Reach here if no authentication policy found in service or namespace level; check for
+	// cluster-scoped (global) policy.
+	// Note: to avoid multiple global policy, we restrict only the one with name equals to
+	// `DefaultAuthenticationPolicyName` ("default") will be used. Also, targets spec, if exists,
+	// is also ignored (if policy is submitted with `istioctl create`` command, this condition will
+	// be validated at submission time)
```

Comment:
We shouldn't rely on `istioctl` specific validation here. For now, validation logic should be encoded in pilot model's common model validation which is also invoked by server-side validation webhook.

Reply:
I don't. I just put the comment to explain why we use only one spec in the next loop and do not check targets. Remove mentions of istioctl to avoid confusion.